packages/schemas/tables/sso_connector_idp_initiated_auth_configs.sql

```diff
@@ -0,0 +1,20 @@
+/** init_order = 2 */
+create table sso_connector_idp_initiated_auth_configs (
+  tenant_id varchar(21) not null 
+    references tenants (id) on update cascade on delete cascade,
+  /** The globally unique identifier of the SSO connector. */
+  connector_id varchar(128) not null
+    references sso_connectors (id) on update cascade on delete cascade,
+  /** The default Logto application id. */
+  default_application_id varchar(21) not null
+    references applications (id) on update cascade on delete cascade,
+  /** OIDC sign-in redirect URI. */
+  redirect_uri text,
```

Comment:
what will happen if we do not specify this `redirect_uri`?

Reply:
Fallback using the first registered one in the application's client metadata.  If both are empty throw an exception when an authentication request comes.